package.json

```diff
@@ -19,17 +19,16 @@
     }
   },
   "scripts": {
-    "test": "npm run test:typescript && npm run test:javascript",
+    "test": "npm run build && npm run test:typescript && npm run test:javascript",
     "test:typescript": "jest",
-    "test:javascript": "npm run test:javascript:dist && npm run test:javascript:legacy",
+    "test:javascript": "npm run build && npm run test:javascript:dist && npm run test:javascript:legacy",
     "test:javascript:dist": "INK_TEST=dist jest --config='jest.config.javascript.js'",
     "test:javascript:legacy": "INK_TEST=legacy jest --config='jest.config.javascript.js'",
     "test:compileFiles": "node src/tests/compile.js",
     "build": "tsc -p ./ && rollup -c",
     "lint": "eslint 'src/**/*.ts'",
     "lint:fix": "npm run lint -- --fix",
-    "watch": "tsc -p ./ && rollup -c -w",
-    "prepare": "npm run build"
```

Comment:
`prepare` was triggered all the time, I suggest we just be redondant in the `run test` steps.